<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>Project Title</title>
  <link rel="icon" href="./favicon.ico" type="image/x-icon">
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@shoelace-style/shoelace@2.16.0/cdn/themes/light.css" />

  <script type="importmap">
    {
      "imports": {
        "three": "https://cdn.jsdelivr.net/npm/three@v0.166.1/build/three.module.js",
        "three/addons/": "https://cdn.jsdelivr.net/npm/three@v0.166.1/examples/jsm/"
      }
    }
  </script>
  
  <script type="module"
    src="https://cdn.jsdelivr.net/npm/@shoelace-style/shoelace@2.16.0/cdn/shoelace-autoloader.js"></script>

  <!-- You can add what components you want to include here  -->
  <link rel="stylesheet" href="../index.css">
  <link rel="stylesheet" href="../components/scroll-to-top/scroll-to-top.css">
  <script src="../components/scroll-to-top/scroll-to-top.js"></script>
  <script src="../components/image/image-component.js"></script>
  <script type="module" src="../components/3d-render_jinuk/render.js"></script>

  <!--  -->
</head>

<body>
  <div class="content">
    <sl-button href="../">Back</sl-button>
 
     
    <h1>1. Life Science Sub-system</h1>

    <div id="section-header-1">
    <h2>1.1 Context of Problem</h2>

    <image-component  tag="image" source="../assets/subsystem_interaction.png" subtitle="sub-system relationship diagram"> </image-component>

    <p> Professional Mars Rovers, including NASA’s Curiosity and Perseverance, deploy powerful but resource-intensive instruments such as X-ray diffraction, 
      laser-induced breakdown spectroscopy and mass spectrometry to identify geochemical and organic markers. Although highly sophisticated, 
      these devices possess a few downsides including: </p>

    <p><ul>
    <li><strong>High Power and Mass:</strong> Spaceflight-grade instruments are large and energy-hungry.</li>
    <li><strong>Ambiguous Biotic vs. Abiotic Signals:</strong> Detecting carbon-based molecules is not always enough to confirm biological origins.</li>
    </ul></p>

    <p> On the other hand, student-Led Mars Rovers often adopt simpler and lower-cost methods such as colorimetric or 
      small-scale optical detection, typically for for proteins or pigments. 
      While more feasible within typical university budgets, these approaches: </p>

    <p><ul>
    <li>Provide narrow or shallow detection scope</li>
    <li>Risk false negatives or false positive as determining the results using naked eyes is relatively unreliable </li>
    <li>May struggle in dusty conditions, as soil coloration can confound results</li>
    </ul></p>

    <p>Hence, two principal value propositions for the life science subsystem were determined in the previous semester as follows:</p>

    <p><ul>
    <li>A comprehensive science plan that ensures high-accuracy detection of potential 
      biosignatures which provides critical insights into the habitability of selected sites that replicate Martian conditions.</li>
    <li>A science plan that enables real-time, in-situ onboard analysis of soil samples for potential life signals with minimal reliance on external technical 
      support which enhances the rover's scientific capability and overall mission value.</li>
    </ul></p>

    <p>
      Building on these value propositions, the following design statements guide the development of a solution that can:
    </p>

    <p>
      Design statement (1): Detect multiple type of biosignatures
    </p>

    <p>
      Design statement (2): Enable real-time detection of potential life markers onboard the Mars rover    
    </p>

    <p>
      Design statement (3): Provide results interpretable by minimal amount of onboard device with a short duration
    </p>

    <p>
    This semester, I continue to develop solutions that closely adhere to the established criteria. However, several iterations have been made to account
      for the engineering complexities involved in constructing the onboard system. In addition, given the high cost and limited availability of the necessary 
      chemicals in the NUS laboratory, my focus in this semester is on refining the chemistry experiments through literature review and only starting to 
      commence hands-on testing in the laboratory once the experimental flow is approved by the engineering team.
    </p>

    <div id="section-header-2">
     <h2>1.2 Concept Development</h2>
      
    <div id = "section-header-2-1">
    <h3>1.2.1 Concept 1: Colorimetric Assays as Stopgap</h3>

    <p>
      As stated in our previous semester’s concept selection, the original plan was to conduct all life detection tests using a spectrofluorometer due to its higher sensitivity and broader detection capability.
      However, as the competition timeline grew tight and the UREX team’s spectrofluorometer was still in development in Semester 1, we began by implementing colorimetric assays. Specifically:
    </p>

    <image-component 
       tag="image" 
       source="../assets/life_science/Table 1.5 Overview of Protein Detection Tests and Their Procedures.png"
       subtitle="Table 1.1 Overview of Protein Detection Tests and Their Procedures">
    </image-component>

    <p>
      These classical laboratory experiments require only minimal instrumentation, particularly reagents, test tubes and a camera to provide direct visual confirmation. 
      We completed initial bench trials and even tested soil samples to ascertain feasibility. 
      These experiments validated that colorimetric assays could serve as a fallback if the advanced instrument was unavailable to ensure basic compliance with URC’s life detection requirements.
    </p>

    <h3>1.2.2 Concept 2: Pivot to Spectrofluorometry</h3>
      
    <p>With the UREX team’s timely delivery of a compact and low-power spectrofluorometer, we reverted to our primary plan:</p>

    <p><ul>
    <li>Integrate the spectrofluorometer into a sealed test chamber for in-situ soil analysis.</li>
    <li>Use fluorescent markers and dyes that respond to various biosignatures.</li>
    </ul></p>

    <p>This pivot drastically increased detection power and consolidated the subsystem design, which allows us to retire colorimetric assays before further testing and iterations.<p>
   <p>For the spectrofluorometric tests, we initially mapped out five key biosignatures for the spectrofluorometer to detect:</p>

      
    <p><ul>
    <li>Proteins: Smaller amino acid fragments survive Martian radiation better but intact proteins reduce false positives by confirming more specific biological origins.</li>
    <li>Nucleic Acids: Strongly life-specific molecules whose partial fragments remain key indicators of past or present biological processes.</li>
    <li>FAD (Flavin Adenine Dinucleotide): Signals active or historic cellular respiration and therefore highlights underlying metabolic activity.</li>
    <li>FDA (Fluorescein Diacetate): Reveals live enzyme activity through fluorescence, which offers a sensitive measure of current microbial viability.</li>
    <li>Enantiomeric Excess: Confirms biotic origins by detecting an imbalance in amino acid chirality that is nearly impossible to replicate through abiotic processes.</li>
    </ul></p>

    <p>
      This comprehensive suite addresses major structural, metabolic and functional markers of life which all measurable by suitable excitation-emission scanning in one spectrofluorometer.<p>

    <p>However, due to time constraints and the complexities of adding specialized reagents for enantiomeric excess detection, we opted to omit this test. 
      While analyzing amino acid chirality could help further minimize false positives, we prioritized URC 2025 eligibility by focusing on our remaining four methods, 
      which already offer a robust and comprehensive approach to life detection.
    </p>

    <h3>1.2.2.1 Concept 2.1</h3>

    <p>
      The detailed procedures for the initial plan of detecting each chosen biosignature using the spectrofluorometer are outlined in the subsections below.
    </p>

      <sl-tree>
        
        <sl-tree-item >
          <a href="./protein.html">Protein</a>
        </sl-tree-item>
        
        <sl-tree-item >
          <a href="./nuclei_acids.html">Nuclei Acid</a>
        </sl-tree-item>
        
        <sl-tree-item>
          <a href="./fad.html">FAD</a>
        </sl-tree-item>
        
        <sl-tree-item>
          <a href="./fda.html">FDA</a>
        </sl-tree-item>
      
  
      </sl-tree> 

    <p>
      The overall experimental procedure is consistent across all tests as shown below: 
    </p>

    <ol>
    <li>Soil is combined with a buffer solution (solution 1).</li>
    <li>The solution with soil is stirred or shaken for five minutes.</li>
    <li>The mixture is then filtered.</li>
    <li>4.	The resulting supernatant is mixed with a fluorescent solution (solution 2) to activate its fluorescent properties.</li>
    <li>The mixture is then stirred or shaken for another five minutes.</li>
    <li>The sample is then sent for fluorescence measurement.</li>
    </ol>

      <image-component 
       tag="image" 
       source="../assets/Table 1.2 Overview of Fluorescence-Based Biosignature Detection Tests for Concept 2.1.png"
       subtitle="Table 1.2 Overview of Fluorescence-Based Biosignature Detection Tests for Concept 2.1">
    </image-component>
      
    <p>
      All tests require distinct buffer and secondary solutions. Hence, it results in a total need for four different buffer solutions and four corresponding second solutions onboard.

      However, because many of the required chemicals were expensive and not readily available from the NUS Chemistry Department, coupled with budget constraints and lengthy lead times, 
      we then conducted another round of research to design experiments using only reagents directly procurable from the NUS Chemistry department laboratory. 
      The result is an iterated version of the spectrofluorometric experiment plan outlined in the next concept.
    </p>

    <h3>1.2.2.2 Concept 2.2</h3>

    <p>
      The detailed procedures for the iterated plan of detecting each chosen biosignature using the 
      spectrofluorometer are outlined in the subsections below.
    </p>
      <sl-tree>
        
        <sl-tree-item >
          <a href="./protein 22.html">Protein</a>
        </sl-tree-item>
        
        <sl-tree-item >
          <a href="./nuclei_acids.html">Nuclei Acid</a>
        </sl-tree-item>
        
        <sl-tree-item>
          <a href="./fad.html">FAD</a>
        </sl-tree-item>
        
        <sl-tree-item>
          <a href="./fda.html">FDA</a>
        </sl-tree-item>
        
        <sl-tree-item>
          <a href="./enantiometric_excess_of_amino_acids.html">Enantiomeric Excess of Amino Acids</a>
        </sl-tree-item>
  
      </sl-tree> 
    <p>
      The overall experimental procedure is similar as concept 2.1 as stated below.
    </p>

    <ol>
    <li>Soil is combined with a buffer solution.</li>
    <li>The solution with soil is stirred or shaken for five minutes.</li>
    <li>The mixture is then filtered.</li>
    <li>The resulting supernatant is mixed with a second solution to activate its fluorescent properties.</li>
    <li>The mixture is then stirred or shaken for another five minutes.</li>
    <li>The sample is then sent for fluorescence measurement.</li>
    </ol>

    <p>
      All tests require distinct buffer and secondary solutions. Hence, it still results in a total need for four different buffer solutions and four corresponding second solutions onboard.
    </p>

    <p>
      However, once the engineering team began designing the science payload and realized the complexity of fitting 8 pumps and 8 beakers, 
      we decided to iterate the experiment plan. By adopting a universal buffer approach, we could significantly reduce the number of onboard reagents. 
      This updated spectrofluorometric experiment plan is presented in the following concept.
    </p>

    <h3>1.2.2.3 Concept 2.3</h3>

    <p>
      The detailed procedures for the iterated plan of detecting each chosen biosignature using the spectrofluorometer and an universal buffer are outlined in the subsections below.
    </p>

    <p>
      The overall experimental procedure is still similar as Concept 2.1 as stated below.
    </p>

    <ol>
    <li>Soil is combined with a buffer solution.</li>
    <li>The solution with soil is stirred or shaken for five minutes.</li>
    <li>The mixture is then filtered.</li>
    <li>The resulting supernatant is mixed with a second solution to activate its fluorescent properties.</li>
    <li>The mixture is then stirred or shaken for another five minutes.</li>
    <li>The sample is then sent for fluorescence measurement.</li>
    </ol>

     <p>
       In this revised concept, all tests use a single universal buffer solution with separate secondary reagents for each assay. By consolidating the buffer requirements, 
       the total onboard solutions are reduced to one common buffer plus five secondary solutions, which both lightens the science payload and simplifies the engineering 
       team’s design of the onboard experiment setup.
     </p>

     <p>
       In conclusion, it is still suggested for the engineering team to design an onboard experiment setup capable of accommodating a total of five experiments. 
       This additional setup would serve as a backup in case of any issues with the primary configuration and ensure readiness for future developments, 
       particularly in testing the enantiomeric excess of amino acids.
     </p>

    <h3>1.3 Testing and Analysis</h3>

    <p>
      Testing and Analysis for Concept 1
    </p>

    <p>
      In Semester 1, we focused on conducting experiments for protein testing, and all three experiments successfully demonstrated the presence of protein. 
      The experiment outcomes are stated below.    
    </p>

    <image-component 
       tag="image" 
       source="../assets/life_science/Figure 1.1 Experimental Setup and Results for Protein Detection Tests (Xanthoproteic Test, Biuret Test, Ninhydrin Test).png"
       subtitle="Figure 1.1 Experimental Setup and Results for Protein Detection Tests (Xanthoproteic Test, Biuret Test, Ninhydrin Test)">
    </image-component>

    <image-component 
       tag="image" 
       source="../assets/life_science/Table 1.6 Summary of Experimental Outcomes for Protein Detection Tests.png"
       subtitle="Table 1.6 Summary of Experimental Outcomes for Protein Detection Tests">
    </image-component>

    <p>
      On top of that, second round of testing was conducted to evaluate the protein detection capability of 
      these methods in the presence of soil. Six test tubes were prepared for each protein detection test with the following configurations:
    </p>

    <image-component 
       tag="image" 
       source="../assets/life_science/Table 1.7 Experiment Settings for Second Round of Protein Detection Testing.png"
       subtitle="Table 1.7 Experiment Settings for Second Round of Protein Detection Testing">
    </image-component>

    <p>
      The key findings are outlined below:
    </p>

    <image-component 
       tag="image" 
       source="../assets/life_science/Figure 1.2 Experimental Setup and Results for Ninhydrin Test.png"
       subtitle="Figure 1.2 Experimental Setup and Results for Ninhydrin Test">
    </image-component>

  <image-component 
       tag="image" 
       source="../assets/life_science/Table 1.8 Summary of Experimental Outcomes for Ninhydrin Test.png"
       subtitle="Table 1.8 Summary of Experimental Outcomes for Ninhydrin Test">
    </image-component>

    <image-component 
       tag="image" 
       source="../assets/life_science/Figure 1.3 Experimental Setup and Results for Biuret Test.png"
       subtitle="Figure 1.3 Experimental Setup and Results for Biuret Test">
    </image-component>

    <image-component 
       tag="image" 
       source="../assets/life_science/Table 1.9 Summary of Experimental Outcomes for Biuret Test.png"
       subtitle="Table 1.9 Summary of Experimental Outcomes for Biuret Test">
    </image-component>

    <image-component 
       tag="image" 
       source="../assets/life_science/Figure 1.4 Experimental Setup and Results for Xanthoproteic Testpng.png"
       subtitle="Figure 1.4 Experimental Setup and Results for Xanthoproteic Test">
    </image-component>

    <image-component 
       tag="image" 
       source="../assets/life_science/Table 1.10 Summary of Experimental Outcomes for Xanthoproteic Test.png"
       subtitle="Table 1.10 Summary of Experimental Outcomes for Xanthoproteic Test">
    </image-component>

    <p>
      Testing and Analysis for Concept 2
    </p>

    <p>
      After the development of the spectrofluorometer is confirmed, we shift our plan to focus on conducting the experiments of Concept 2.
    </p>

    <image-component 
       tag="image" 
       source="../assets/PT stacked 1.png"
       subtitle="Table 1.10 Summary of Experimental Outcomes for Xanthoproteic Test">
    </image-component>

    <image-component 
       tag="image" 
       source="../assets/PT overlay.png"
       subtitle="Table 1.10 Summary of Experimental Outcomes for Xanthoproteic Test">
    </image-component>

    <image-component 
       tag="image" 
       source="../assets/DNA stacked 1.png"
       subtitle="Table 1.10 Summary of Experimental Outcomes for Xanthoproteic Test">
    </image-component>

    <image-component 
       tag="image" 
       source="../assets/DNA overlay.png"
       subtitle="Table 1.10 Summary of Experimental Outcomes for Xanthoproteic Test">
    </image-component>

    <image-component 
       tag="image" 
       source="../assets/FDA stacked 1.png"
       subtitle="Table 1.10 Summary of Experimental Outcomes for Xanthoproteic Test">
    </image-component>

    <image-component 
       tag="image" 
       source="../assets/FDA overlay.png"
       subtitle="Table 1.10 Summary of Experimental Outcomes for Xanthoproteic Test">
    </image-component>

    <image-component 
       tag="image" 
       source="../assets/FAD stacked 1.png"
       subtitle="Table 1.10 Summary of Experimental Outcomes for Xanthoproteic Test">
    </image-component>

    <image-component 
       tag="image" 
       source="../assets/FAD overlay.png"
       subtitle="Table 1.10 Summary of Experimental Outcomes for Xanthoproteic Test">
    </image-component>

    <image-component 
       tag="image" 
       source="../assets/PT stacked 2.png"
       subtitle="Table 1.10 Summary of Experimental Outcomes for Xanthoproteic Test">
    </image-component>

    <image-component 
       tag="image" 
       source="../assets/DNA stacked 2.png"
       subtitle="Table 1.10 Summary of Experimental Outcomes for Xanthoproteic Test">
    </image-component>

    <image-component 
       tag="image" 
       source="../assets/FDA stacked 2.png"
       subtitle="Table 1.10 Summary of Experimental Outcomes for Xanthoproteic Test">
    </image-component>

    <image-component 
       tag="image" 
       source="../assets/FAD stacked 2.png"
       subtitle="Table 1.10 Summary of Experimental Outcomes for Xanthoproteic Test">
    </image-component>


    <h3>1.4 Fulfilment of deliverables</h3>
 <p>The proposed experiment successfully addresses all the outlined design statements through a compact and efficient life detection strategy. First, it fulfills Design Statement 1 by targeting a diverse range of biosignatures—specifically amino acids, DNA/RNA, flavin adenine dinucleotide (FAD), and fluorescein diacetate (FDA). These markers were selected based on their biological relevance and detectability under Martian conditions, thus enabling comprehensive screening for multiple types of potential life-related molecules.
    </p>
 <p>In alignment with Design Statement 2, the experiment is designed to support real-time detection onboard the Mars rover. The use of fluorescence-based detection allows for rapid spectral acquisition, with each measurement requiring less than one minute. This fast response time enables efficient data collection during field operations, minimizing delays and maximizing mission productivity.
    </p>
<p>Lastly, the setup satisfies Design Statement 3 by being both equipment-light and time-efficient. The entire workflow requires only a single spectrofluorometer, significantly reducing the demand for bulky instrumentation and power consumption onboard the rover. Additionally, the complete detection process—from sample preparation to fluorescence reading—can be completed in under 20 minutes, ensuring timely and interpretable results without overburdening the rover’s processing capabilities.
    </p>

    <h3>1.5 Shortcomings and Future Work</h3>
 <p>One key limitation observed in the current experimental setup is the fluorescence spectral overlap between DNA/RNA and the buffer solution. The characteristic peaks of DNA/RNA are found to be too close to those of the buffer, which increases the risk of false positives and may compromise the reliability and accuracy of result interpretation. To mitigate this issue, extensive repeat testing is required to validate the consistency of the fluorescence signals obtained. Future work could involve exploring alternative detection strategies or modifying the buffer formulation to enhance peak distinction and reduce background interference.</p>

 <p>Another potential shortcoming arises from the complex composition of Martian soil or regolith. The presence of minerals or perchlorates in the soil may either exhibit autofluorescence or act as quenching agents, leading to false positive signals or suppression of true fluorescence responses. This introduces additional challenges to signal clarity and biosignature identification. As part of future efforts, it is essential to expand the range of potential interference testing to include soil compositions that closely mimic Martian conditions. Investigating the interaction of the proposed experimental approach with known Martian analog materials can help refine detection reliability and support the development of more robust signal validation protocols.
</p>
      
    <h1>BELOW IS ALL OLD STUFF</h1>

      

      
      
    <p>After confirming the selected biosignatures for detection, the next step involves determining the appropriate detection method. A selection matrix
       shown in Table 1.4 was employed to evaluate the capability of each detection method for identifying the chosen biosignatures, where a score of 1 
       indicates the method can detect the biosignature and 0 indicates it cannot.</p>

       <image-component 
       tag="image" 
       source="../assets/life_science/Table 1.4 Selection Matrix Table for Analyzing Devices Based on Biosignature Detection Capability.png"
       subtitle="Table 1.4 Selection Matrix Table for Analyzing Devices Based on Biosignature Detection Capability">
       </image-component>

    <p>As shown in Table 1.4, the spectrofluorometer was able to detect all five targeted biosignatures. Hence, the spectrofluorometer is selected as the 
      primary instrument for our science module to conduct all experiments related to detecting the chosen biosignatures. This choice aligns with the goal 
      of addressing the research gaps identified in the problem context, specifically the need for a compact and energy-efficient design for Mars missions. 
      The spectrofluorometer's ability to perform multiple tests using a single, versatile instrument ensures minimal yet highly effective technical instrumentation,
       contributing to the sustainability and operational portability of the Mars rover's scientific payload.</p>

    </div>

    <div id="Detailed procedures">
      <h3>Detailed procedures</h3>
  
      <p>The detailed procedures for detecting each chosen biosignature using the spectrofluorometer are outlined in the subsections below.</p>
      
    </div> 
    
    <div id = "section-header-2-2">
    <h3>1.2.2 Concept 2</h3>

    <p>However, given that our Mars rover is being developed for the University Rover Challenge 2025, priority is placed on meeting the participation requirements
      for the science mission due to constraints in manpower and time. Therefore, the highest-ranked biosignatures in the selection matrix, Protein will be prioritized
      for testing without relying on external technology. This decision is made due to the fact that the development of the spectrofluorometer is managed by the UREX 
      group, which introduces uncertainty regarding its availability and successful development.
    </p>

    <p>Hence, a series of protein detection tests, independent of any external analytical devices, is conducted to make a comprehensive decision on the final selection 
      of the onboard selection. The Ninhydrin Test, Biuret Test, and Xanthoproteic Test were selected because each targets distinct characteristics and functional groups
      within proteins. This approach enables a more comprehensive analysis, as the outcomes of these tests can complement one another to enhance both the accuracy and 
      sensitivity of the results. The detailed of each detections test are outlined in Table 1.5 below.
    </p>



    </div>

    <div id = "section-header-3">
    
    <h2>1.3 Testing and Analysis</h2>

    <p>The initial phase of testing focuses on evaluating the protein detection capability. Each test includes two test tubes: one containing the protein sample and the
      other serving as a control. The protein sample is prepared by dissolving protein powder in ultrapure water, while the control contains only ultrapure water. The
      setup prior to adding the protein detection reagents is depicted in the Figure 1.1 on the left. Following the completion of all protein detection tests, the 
      expected outcomes are outlined in the "Observation for Positive Result" column of Table 1.5. The actual experimental results are displayed in Figure 1.1 on the right. 
    </p>


    
    <p>A summary and analysis of the experimental outcomes are provided in Table 1.6 below.</p>


    <p>As shown in the summary table, all three protein detection tests demonstrated strong efficacy in detecting the presence of proteins. Consequently, given that
       the life detection tests in the URC Science Mission will be performed with soil samples, the second round of testing will evaluate the protein detection capability
        of these methods in the presence of soil.
    </p>

    <p>In the second round of testing, six test tubes are prepared for each protein detection test with the following configurations in Table 1.7.</p>



    <p><strong>Objectives:</strong>
    <ul>
    <li>Test Tubes 1, 2, 3, 4, and 5 are used to assess the effect of soil on protein detection.</li>
    <li>Test Tubes 1 and 2, and Test Tubes 3 and 4 are used to evaluate whether stirring influences the outcome of the experiment.</li>
    </ul>
    </p>

    <p>The hypothesis for this round of testing is that the presence of soil and the method of mixing (stirring versus no stirring) will influence the protein 
      detection outcomes. It is anticipated that stirring will improve the distribution of the protein within the soil, enhancing detection efficiency, while 
      the soil itself may introduce potential interferences that could affect the accuracy of the results. This test aims to determine whether soil particles or
       the absence of stirring could lead to false negatives or reduced detection capability.
    </p>

    <p>The setup before adding the protein detection reagents is shown in the image on the left in each figure from Figure 1.2 to Figure 1.4. After completing the
       protein detection tests, the anticipated positive outcomes are indicated in Figure 1.2 to Figure 1.4 with red-coloured text. The actual experimental results
        are displayed in the image on the right of Figure 1.2 to Figure 1.4. A summary and analysis of the experimental outcomes are provided in the table right after
         each figure, from Table 1.8 to Table 1.10.
    </p>



  
    </div>

    <div id = "section-header-4">

    <h2>1.4 Shortcomings Identified</h2>

    <p>The experiments conducted using the Ninhydrin, Biuret, and Xanthoproteic Tests demonstrated that protein detection in the presence of soil 
      is feasible, though its effectiveness varies due to factors such as mixing and colour interference from soil particles. To address these limitations,
        the inclusion of stirring and filtration steps is considered. The experimental results showed that stirring facilitated the even distribution of 
        proteins within the solution, ensuring consistent interaction with the reagents. Additionally, filtration are considered because it is known to be
        beneficial in creating a clearer solution by removing particulate matter that could obscure the results, thereby enhancing the accuracy and ease of
      observing colour changes (Jacobsen, 2004). Therefore, it shows a necessity for the mechanical subsystem, overseen by Jian Ling, to develop an 
      effective stirring and filtration mechanism.
    </p>

    </div>

    <div id = "section-header-5">

    <h2>1.5 Future Work</h2>  

    <p>To ensure the consistency and reliability of the experiment outcomes, further tests on Ninhydrin, Biuret, and Xanthoproteic Tests incorporating stirring and filtration will be conducted in the lab. These additional trials will help validate the effectiveness of these steps in enhancing protein detection accuracy in soil samples.</p>

    <p>Meanwhile, I will shift my focus to conducting the experiments detailed in Section 1.2.1. Concept 1, which involve the use of a spectrofluorometer. This approach aligns to our value proposition which aims not only to fulfil the requirements for developing a science module that meets the participation criteria for URC 2025 but also to advance beyond basic organic detection. The goal is to set a new benchmark in astrobiology and life detection on Mars by addressing the research gaps identified in the “1.1 Context of Problem” section.</p>

    </div>


    </div>

     <sl-button href="../">Back to Main</sl-button>
     <sl-button href="../vision/">Next Subsystem</sl-button>
     <sl-button class="scroll-to-top" variant="primary" size="medium" circle onclick="scrollToTop()">
      <sl-icon name="arrow-up" label="Settings"></sl-icon>
    </sl-button>
</body>

</html>
